GSA-TTS/FAC
# Notes to SEFA Upload Page

# Notes to SEFA Upload Page & Checklist Restructure

Issue: https://github.com/GSA-TTS/FAC/issues/1560
Figma: [Link](https://www.figma.com/file/lnkXJTfKzD3YBhiJZ2QFEv/FAC-final?node-id=197%3A6148&mode=dev)

## Changes:

1. Notes to SEFA (NTS) is upload-able
    * A new upload page for NTS
    * A place for NTS in the checklist
    * A removed line from the checklist handler that allows NTS to be part of the checklist state
2. Checklist changes
    * Slight reordering of workbooks. Audit info and report PDF upload are above the workbooks
3. Upload page changes
    * A back button
    * A link in the instructions that goes to the workbook page on the static site
4. A new sting in `config.settings` for holding the static site base URL
    * Can be updated as we choose new URL's
    * Can possibly be made into an environment variable if things differ too much between environments
5. Cypress - test for notes to SEFA workbook upload

## How to test

Switch to this branch and run normally. Navigate to or create a submission. Verify that the new upload page is present, and that the checklist works as expected given the new element (and ordering). 

## PR checklist: submitters


## PR checklist: reviewers

        
The larger the PR, the stricter we should be about these points.
